### ISSUE TITLE
[Bug]: Randomly invisible tamed dog.

### ISSUE THREAD

**Issue by DigitalWendigo** [someaddons/performant_issues#234]
### Describe the bug you're experiencing

My tamed dog (vanilla) randomly becomes invisible. 
It can happen at any moment, usually while walking through different chunks. 
Now I also have optifine installed and entity culling. 
I've consulted the mod maker of Entity culling and they said it's probably to do with performant. 
I've also consulted optifine and awaiting answers. 
I also have Radon installed, which changes the games lighting engine.
And Ferrite Core for RAM optimization.
Amongst those I also have some minor mods installed that alter the playstyle. 
These include Create and it's dependant Flywheel.
There are other ones as well but these do not affect rendering, entity culling, altering AI or altering models in any way or form. 

### Reproducability

Use latest Performant version. 
Have the following mods installed.
-Radon
-OptiFine HD_U_G8
-Entity Culling
-Create + Flywheel
-Ferrite Core
-LazyDFU

Play the game , tame a dog, walk and run around different chunk sand eventualy have an invisible tamed dog follow you. 

### Mod up to date

- [X] Before submitting this issue I updated to the newest version and reproduced it

### Minecraft version

1.16.5

### Modloader version

Forge

### Logs

https://gist.github.com/DigitalWendigo/b7dfaea162a20cf3b20ca41dc5d25f11

**Comment by someaddons** [comment]
can you reproduce it without performant?

**Comment by DigitalWendigo** [comment]
Sorry that it's taken so long to reply, I didn't get a notification from Github that you had replied to my issue, for whatever reason. 
I've done several tests and managed to find the culprit. 
The issue seems to happen when Entity culling by tr9zw is paired with performant and optifine and support regarding the issue has been dropped because the author said this.
"Performant is not tested at all, pretty sure it's because of that. Not planning to add support for it, since it's just old forge versions."
I strongly believe performant isn't to blame since at least you provide help and support, and potential fixes when kindly asked. 
So I've just removed entity culling and the issues have been resolved entirely.
